### PR TITLE
Training

### PR DESCRIPTION
**Pull request by ttrine**
Substantial refactoring:
- Remove unnecessary files
- Remove data folder to reflect the nonlocality of our data
- Move game_converter and preprocessing out of models, into new folder called preprocessing
- Rename training scripts for consistency

All tests are passing; all imports have been updated to reflect the changes in folder structure.

I'm submitting a pull request because I intend this as a proposal. I think these changes clarify and simplify our directory structure for newcomers. If you would like to ask me about any of these changes, or suggest revisions, we can have the discussion here.

### CODE REVIEW

File: AlphaGo/models/policy.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from keras.layers import convolutional
 from keras.layers.core import Activation, Reshape
 import keras.backend as K
-from preprocessing import Preprocess
+from AlphaGo.preprocessing.preprocessing import Preprocess
```
Comment (wrongu) [inline AlphaGo/models/policy.py]: this line makes me think we should rename things or further move them around

**Comment by jtauber** [comment]
This PR has a few style/whitespace changes unrelated to the core refactoring. This is one reason why having a PEP8 cleanup (and subsequent flake8 check in travis-ci) can be very worthwhile.

**Comment by ttrine** [comment]
That makes sense. I tried to make as few unrelated changes as possible but
I am still learning! Is a PEP8 cleanup a manual process?

File: AlphaGo/models/policy.py (hunk 1 above)
Comment (ttrine) [inline AlphaGo/models/policy.py]: I was thinking that too. Both modules in this folder handle different aspects of preprocessing. What if we merged the code from game_converter into preprocessing and let it live in /AlphaGo? Would that be too bloated?

**Comment by jtauber** [comment]
@ttrine some of the changes were likely just your editor automatically doing them (e.g. stripping trailing whitespace)

I've got a ticket to manually fix the violations and then hook up a tool, flake8, to check we don't regress but not all violations are equal and some might be controversial, especially if you're coming from a different language :-)

**Comment by jtauber** [comment]
The key is really just for a project to have a consistent code style that everyone follows so commits don't make arbitrary style changes in addition to what's really being meant to be contributed.

**Comment by jtauber** [comment]
(and sorry, I didn't mean to hijack this PR; was just relevant. followups to #44 )

**Comment by wrongu** [comment]
Note: the latest travis error is an esoteric compiler bug having to do with specifying the right CPU architecture for Theano to use in the travis environment. Tests pass locally, so we are merging..

**Pull request merged**